src/openforms/submissions/api/serializers.py

```diff
@@ -276,19 +276,27 @@ class SubmissionSuspensionSerializer(serializers.ModelSerializer):
             "The email address where the 'magic' resume link should be sent to"
         ),
     )
+    suspend_allowed = serializers.BooleanField()
 
     class Meta:
         model = Submission
         fields = (
             "email",
             "suspended_on",
+            "suspend_allowed",
         )
         extra_kwargs = {
             "suspended_on": {
                 "read_only": True,
             }
         }
 
+    # TODO: Move validators to view?
+
+    def validate_suspend_allowed(self, suspend_allowed):
+        if not suspend_allowed:
+            raise exceptions.PermissionDenied(_("Suspending this form is not allowed."))
+
```

Comment:
Yeah, I think this check belongs in the `_suspend` view, right after loading the submission and before de-serialising the user input. The fact that it doesn't raise a `ValidationError` is a tell tail sign.